README.md

```diff
@@ -1158,6 +1158,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [WunderGraph](https://cloud.wundergraph.com) - An open-source platform that allows you to  quickly build, ship and manage modern APIs. Built-in CI/CD, GitHub integration, and automatic HTTPS. Up to 3 projects, 1GB egress, 300 minutes of build time per month on the [free plan](https://wundergraph.com/pricing)
   * [Zeabur](https://zeabur.com) - Deploy your services with one click. Free for three services, with US$ 5 free credits per month.
   * [mogenius](https://mogenius.com) - Easily build, deploy, and run services on Kubernetes. The free tier supports connecting a local Kubernetes with mogenius, enabling individual developers to create a production-like test environment on their machine.
+  * [genezio](https://genezio.com/) - Serverless function provider that gives 1 million function invocations, 100GB bandwidth, and 10 cron jobs per month.
```

Comment:
Looks like this is non commercial and academic use only for free? Please mention that